homeassistant/components/mqtt/sensor.py

```diff
@@ -210,7 +210,7 @@ async def async_will_remove_from_hass(self) -> None:
         """Remove exprire triggers."""
         # Clean up expire triggers
         if self._expiration_trigger:
-            _LOGGER.debug("Clean up expire after trigger for %s", self.entity_id)
+            _LOGGER.info("Clean up expire after trigger for %s", self.entity_id)
```

Comment:
```suggestion
            _LOGGER.debug("Clean up expire after trigger for %s", self.entity_id)
```

Reply:
In that case I will need to remove the assetions from the tests.